testdata/26-testbenchmultiple.txt

```diff
@@ -0,0 +1,23 @@
+=== RUN   TestRepeat
```

Comment:
This test appears to be a combination of tests 24 and 25, so it probably doesn't add much value. Can you instead change it so this tests benchmarks from multiple packages. For example, something you would see when you run `go test -v -bench . ./...` in a directory that contains 2 separate packages each with a benchmark? Because I don't think that case is currently handled correctly by the parser.